### ISSUE TITLE
Route `/list` does not work with responsive UI

### ISSUE THREAD

**Issue by chakflying** [louislam/uptime-kuma#359]
**Describe the bug**
the "/list" route doesn't work with responsive UI.

**To Reproduce**
1. Login to dashboard, resize browser to mobile view
2. Click list to go to List page
3. Resize back to desktop view
4. Observe duplicate list view
![image](https://user-images.githubusercontent.com/3271800/132616218-35d97bc6-d6e8-4c83-8e8d-00cec5cfc154.png)

**Expected behavior**
I'm not sure how we should handle this either.

**Info**
- Uptime Kuma Version: master
- Using Docker?: No
- OS: Windows 10 21H1
- Browser: Firefox 91.0.2

**Comment by louislam** [comment]
Yes, this route is actually for mobile only. May need to redesign or route to homepage in desktop.

**Comment by sanjogbhalla16** [comment]
I am interested in working upon this , can you please guide me , how to do it , it is my first time contribution
